### DISCUSSION TITLE
500 server error when loging in

### DISCUSSION THREAD

**Discussion by jiwidi** [yhype/yhype-feedback#15] [Q&A]
Hi! I get this when loging back with my github, any idea why?

**Comment by antonkomarev** [comment]
@jiwidi try again, please.

**Comment by kitswas** [comment]
The same happens with my account.
![image](https://user-images.githubusercontent.com/90329875/204436296-b782589e-7711-45a0-a9ca-4658ae3c07a8.png)

**Reply by antonkomarev** [reply]
Thanks for the report. Fixed it couple of hours ago.

**Reply by kitswas** [reply]
> Thanks for the report. Fixed it couple of hours ago.

It's happening again today.

**Reply by antonkomarev** [reply]
Fixed. We need to rewrite some logic. RAM leaking :(

**Comment by antonkomarev** [comment]
We found 2 issues and resolved them. Should not return 500 now. 2 days without issues. Thanks for reporting!
